src/stylesheets/components/_navigation.scss

```diff
@@ -11,6 +11,69 @@ $sliding-panel-width: "card-lg";
 $nav-link-accordion-icon-size: 2.5;
 $nav-link-arrow-icon-size: 2;
 
+// Maps for icon masks
+$place-icon: (
+  "color": currentColor,
+  "svg-height": 24,
+  "svg-width": 24,
+);
+
+$chevron-icons: map-merge(
+  $place-icon,
+  (
+    "height": units($nav-link-arrow-icon-size),
+  )
+);
+
+$mobile-icons: map-merge(
+  $place-icon,
+  (
+    "height": units($nav-link-accordion-icon-size),
+  )
+);
+
+$add-icon: map-merge(
+  $mobile-icons,
+  (
+    "name": "add",
+  )
+);
+
+$remove-icon: map-merge(
+  $mobile-icons,
+  (
+    "name": "remove",
+  )
+);
+
+$close-icon: map-merge(
+  $mobile-icons,
+  (
+    "name": "close",
+  )
+);
+
+$expand-more-icon: map-merge(
+  $chevron-icons,
+  (
+    "name": "expand_more",
+  )
+);
+
+$expand-less-icon: map-merge(
+  $chevron-icons,
+  (
+    "name": "expand_less",
+  )
+);
+
+// Maps for place-icon mixin
+$contrast-bg: "default";
+$after: "after";
+$before: "before";
+$icon-margin: "auto";
+$vertical-align: "auto";
```

Comment:
I don't believe`vertical-margin: auto` has any meaning. Do you mean `middle`?

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align